resources/metadata/sonar.yaml

```diff
@@ -5,6 +5,14 @@ metadata:
 spec:
   inputs:
     params:
+      - name: instance
```

Comment:
Where is `instance` used? I don't see it added in the options/config handling.

Reply:
This was the best way to use parameters in groovy without changing the whole `contextConfig` resolution.